.gitignore

```diff
@@ -4,7 +4,6 @@
 # Ignore some folders
 node_modules
 /static/framework
-dist
```

Comment:
Wrong `.gitignore`, I was referring to `framework/molecules/comments/.gitignore` 😉